op/opstatus.go

```diff
@@ -106,19 +108,28 @@ func (s *Status) Check() HealthResult {
 		CheckResults: make([]healthResultEntry, len(s.checkers)),
 	}
 
-	for i, checker := range s.checkers {
-		var cr CheckResponse
-		checker.checkFunc(&cr)
-		hr.CheckResults[i] = healthResultEntry{
-			Name:   checker.name,
-			Health: cr.health,
-			Output: cr.output,
-			Action: cr.action,
-			Impact: cr.impact,
-		}
-		s.updateCheckMetrics(checker, cr)
+	wg := &sync.WaitGroup{}
```

Comment:
This can just be `var wg sync.WaitGroup` which I would say is more idiomatic. (see usage throughout the standard library)